src/main/webapp/feedback.jsp

```diff
@@ -0,0 +1,88 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%
+  boolean canOpen = (boolean) request.getAttribute("feedbackOpen");
+  String role = (String) request.getAttribute("role");
+  pageContext.setAttribute("feedbackOpen", canOpen);
+  pageContext.setAttribute("role", role); 
+%>
+
+
+<c:choose>
+  <c:when test= "${canOpen == false}">
+    <h2 style="text-align: center">You may not submit feedback yet.</h2>
+  </c:when>
+  <c:otherwise>
+    <c:choose>
+      <c:when test= "${role == 'Interviewee'}">
+        <h1 class="text-center">Feedback</h1>
+        <h4 class="text-center">Please submit your feedback for your interviewer below</h4>
```

Comment:
Not a fan of how this is so big compared to the title, same for the h5 you have in the form. Please also get these 2 subtitles in the same box.
Can you style it like the top of this? https://interview-me-step-2020.appspot.com/availability.html

Reply:
Is the way I have it now better?

Reply:
Yeah, just put periods at the ends of sentences please.